lib/import.js

```diff
@@ -0,0 +1,157 @@
+const program = require( 'commander' )
+const sqlite = require( 'sqlite' )
+
+const { Lines, Shabads, Sources, TranslationSources, Translations, knex } = require( '../' )
+
+const { generateId } = require( './utils' )
+require( './string-colors' )
+
+const columnOpts = [ 'shabadId', 'translation', 'page', 'line', 'sttm', 'gurmukhi', 'orderBy', 'source' ]
+
+const findIds = async ( model, length, count ) => {
+  // Grab ids to exclude from database
+  const ids = new Set( ( await model.query().select( 'id' ) ).map( ( { id } ) => id ) )
+
+  const newIds = []
+  while ( newIds.length < count ) {
+    const id = generateId( length )
+    if ( !ids.has( id ) ) {
+      ids.add( id )
+      newIds.push( id )
+    }
+  }
+
+  return newIds
+}
+
+const main = async () => {
+  program
+    .version( '0.0.1' )
+    .arguments( '<input> <tableName>' )
+    .option( '-o --order-by <column>', 'order column name' )
+    .option( '-s --shabad-id <column>', 'Shabad ID column name' )
+    .option( '-S --source <column>', 'Source column name', -1 )
+    .option( '-2 --sttm-id <column>', 'STTM2 Shabad ID column name' )
+    .option( '-t, --translation <column>', 'column name of a translation', ( val, prev ) => [ ...prev, val ], [] )
```

Comment:
Formatting: this and next lines have a comma `,` after `-x` parameter but lines above do not.

Reply:
Unsure what you mean

Reply:
![image](https://user-images.githubusercontent.com/14130567/57963018-6003fd80-78ec-11e9-8a45-66c2ac04425f.png)

Reply:
Oh, I see now, good catch!